lib/IRGen/IRGen.cpp

```diff
@@ -176,22 +176,24 @@ void swift::performLLVMOptimizations(IRGenOptions &Opts, llvm::Module *Module,
                            addSwiftContractPass);
   }
 
-  if (Opts.Sanitize == SanitizerKind::Address) {
+  if (Opts.Sanitizers & SanitizerKind::Address) {
     PMBuilder.addExtension(PassManagerBuilder::EP_OptimizerLast,
                            addAddressSanitizerPasses);
     PMBuilder.addExtension(PassManagerBuilder::EP_EnabledOnOptLevel0,
                            addAddressSanitizerPasses);
   }
   
-  if (Opts.Sanitize == SanitizerKind::Thread) {
+  if (Opts.Sanitizers & SanitizerKind::Thread) {
     PMBuilder.addExtension(PassManagerBuilder::EP_OptimizerLast,
                            addThreadSanitizerPass);
     PMBuilder.addExtension(PassManagerBuilder::EP_EnabledOnOptLevel0,
                            addThreadSanitizerPass);
   }
 
   if (Opts.SanitizeCoverage.CoverageType !=
-      llvm::SanitizerCoverageOptions::SCK_None) {
+      llvm::SanitizerCoverageOptions::SCK_None ||
+      (Opts.Sanitizers & SanitizerKind::Fuzzer)) {
```

Comment:
I see the one in IRGenOptions being set here: https://github.com/apple/swift/pull/9450/files#diff-487c2f92307618024a8f8f553589405dR1681